utils/crypto/bls/signer/localsigner/bls_test.go

```diff
@@ -120,19 +131,18 @@ func TestSingleKeyAggregation(t *testing.T) {
 
 func TestIncorrectMessageAggregation(t *testing.T) {
 	require := require.New(t)
-	sk1, pk1 := NewKeyPair(require)
-	sk2, pk2 := NewKeyPair(require)
-	sk3, pk3 := NewKeyPair(require)
-
-	pks := []*bls.PublicKey{pk1, pk2, pk3}
+	signers := []*LocalSigner{NewSigner(require), NewSigner(require), NewSigner(require)}
+	pks := Map(signers, func(signer *LocalSigner) *bls.PublicKey {
+		return signer.PublicKey()
+	})
```

Comment:
This applies to a number of areas, but if we are going functional:
```suggestion
	pks := Map(signers, (*LocalSigner).PublicKey)
```